packages/hoppscotch-backend/src/infra-config/infra-config.service.ts

```diff
@@ -146,11 +185,23 @@ export class InfraConfigService implements OnModuleInit {
     if (E.isLeft(isValidate)) return E.left(isValidate.left);
 
     try {
+      const dbInfraConfig = await this.prisma.infraConfig.findMany({
+        select: { name: true, isEncrypted: true },
+      });
```

Comment:
Same here too, can't we use `getDefaultInfraConfigs` instead.

Reply:
`getDefaultInfraConfigs` has a DB call.